pkg/controller/nodes/task/resourcemanager/resourceconstraints.go

```diff
@@ -0,0 +1,25 @@
+package resourcemanager
+
+import (
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+	pluginCore "github.com/lyft/flyteplugins/go/tasks/pluginmachinery/core"
+)
+
+type ComposedResourceConstraint struct {
```

Comment:
nit: docs
also, what does the 'composed' mean in the name?
aslo, can you rename the file to match the type name?